arangosh/Shell/ClientFeature.cpp

```diff
@@ -44,17 +45,21 @@
 #include <iostream>
 #include <thread>
 
+#include <Logger/LogMacros.h>
+
```

Comment:
Is this include still needed, or was it just needed for some previous debugging?

Reply:
there's LOG_TOPIC here